packages/nx-plugin/src/executors/internal/types.ts

```diff
@@ -20,6 +20,13 @@ export type ProjectExecutorOnlyOptions = {
  * CLI types that apply globally for all commands.
  */
 export type GlobalExecutorOptions = {
+  command?:
+    | 'collect'
+    | 'upload'
+    | 'autorun'
+    | 'print-config'
+    | 'compare'
+    | 'history';
```

Comment:
This list is missing the `merge-diffs` command (used by GitHub Action in monorepo mode).

```suggestion
  command?:
    | 'collect'
    | 'upload'
    | 'autorun'
    | 'print-config'
    | 'compare'
    | 'merge-diffs'
    | 'history';
```